CONTRIBUTORS.yaml

```diff
@@ -296,6 +296,11 @@ torhou:
 tseemann:
     name: Torsten Seemann
 
+usegalaxyp:
+    name: Pratik Jagtap
```

Comment:
You already have an entry here: https://github.com/galaxyproject/training-material/pull/876/files#diff-cea3975e6d64742067d7fde4e7f37b6cR247